src/android-native/src/main/java/io/liteglue/SQLitePlugin.java

```diff
@@ -52,14 +52,12 @@
     static SQLiteConnector connector = new SQLiteConnector();
 
     static String LOG_TAG = SQLitePlugin.class.getSimpleName();
-
-    protected Activity activity = null;
     protected ExecutorService threadPool;
+    private Context context;
 
-    public SQLitePlugin(ReactApplicationContext reactContext, Activity activity) {
+    public SQLitePlugin(ReactApplicationContext reactContext) {
```

Comment:
As I mentioned, this create unnecessary backward incompatibility. Instead of removing original constructor, I think it will be more appropriate to add a new constructor taking only reactContext. This way both modes of creation are supported.